src/main/java/com/builtamont/cassandra/migration/internal/dbsupport/SchemaVersionDAO.kt

```diff
@@ -309,6 +309,10 @@ open class SchemaVersionDAO(private val session: Session, val keyspaceConfig: Ke
             )
         }
 
+        if (migrationVersions.size == 0) {
+            return 1;
+        }
+
```

Comment:
This could be rewritten as `if (migrationVersions.isEmpty()) return 1` guard clause.

Reply:
Doesn't this line prevent batch from being executed with statements `return i + 1`?